### ISSUE TITLE
Did you solved it

### ISSUE THREAD

**Issue by kopax** [jedwards1211/istanbul-lcov-issue#2]
In a very large reactjs module, I have the same error using jest 18.0.0 and using webpack 2.0.

The error happen if this line is present : 

    export { default as Progress, ProgressBar } from './components/Progress';

Every module within Progress as been tested and the component work, I don't know why I have this error now coming.

**Comment by jedwards1211** [comment]
I don't know if it's been fixed, I created this repo for the istanbul team to look at.

It's hard to know if you're seeing the error for the same reasons as this repo.  I was getting it because I was changing the code in a .js file during testing.  You're not doing something like that are you?

**Comment by kopax** [comment]
No I am not, sorry for disturbing you for that.
I got my answer and it was a bit more tricky.

My test was scanning  for `*.test.js` pattern.

I had this directory tree that was planting istanbul.

I had a Component with the same name as a directory (I was rewriting the component), I don't know why but even if this component wasn't imported anymore, because of the way the report page is build it ended up in a loop throwing this error.

I noticed this by checking the two missing pages in the coverage, deleting my unused components solved my issue.
